client/src/i18n/fr/barcode.json

```diff
@@ -1,5 +1,11 @@
-{"BARCODE":{"SCAN":"Scanner le Code à Barres",
-"AWAITING_INPUT":"En Attente d'Entrée",
-"AWAITING_HTTP":"Code à barres lu! Recherche de factures ....",
-"READ_SUCCESS":"Facture Trouvée",
-"READ_ERROR":"Code à barres illisible! Entrez le code à barres manuellement."}}
\ No newline at end of file
+{
+  "BARCODE" : {
+    "SCAN" : "Scanner le Code à Barres",
+    "AWAITING_INPUT" : "En Attente d'Entrée",
+    "AWAITING_HTTP" : "Code à barres lu! Recherche l'enregistrement....",
+    "READ_SUCCESS" : "Trouvée",
+    "READ_ERROR" : "Code à barres illisible! Entrez le code à barres manuellement.",
```

Comment:
for the "READ_ERROR" a good key and message can be :
```
"READ_FAILED" : "Enregistrement non trouvé"
```
and for any errors during the reading : 
```
"READ_ERROR" : "Une erreur est survenue lors de la lecture du code barre"
```

Reply:
That is an excellent idea.  I'll expand the API for it.